### ISSUE TITLE
Favourite disabling doesn't work

### ISSUE THREAD

**Issue by beckettkev** [beckettkev/Goldfish#3]
Favourite disabling doesn't work when selected from the settings page.

**Comment by jquintozamora** [comment]
Hi Kev, when Favourite is disabled, what should happens? Just hide the favorites tab?

**Comment by beckettkev** [comment]
Yes. So basically we should feed that into the menu component and the
person one so that:

-the button isn't displayed in the people card
-the favourite menu tab is not displayed
